plone/app/upgrade/v50/tests.py

```diff
@@ -179,7 +178,7 @@ def test_fix_double_smaxage(self):
 
 
 def test_suite():
-    # Skip these tests on Plone 4
+    # Skip these tests if not Plone 5.0 or 5.1
```

Comment:
A bit out of scope here, but using ``unittest.skipIf`` https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures would be more elegant here.

Reply:
Yes indeed, thats elegant.